roles/branch-validation/tasks/api.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: check nova
+  shell: "curl -I --connect-timeout 2 http://127.0.0.1:{{ endpoints[item].port.backend_api }}{{ '/' + endpoints[item].version|default('') }}"
```

Comment:
This looks like it'll directly connect to the service, rather than hitting the floating IP address and then haproxy and letting haproxy decide what backend to hit.

I think we want to emulate what a customer would see, which is floating IP -> haproxy -> service.